jdroid-gradle-android-plugin/src/main/groovy/com/jdroid/gradle/android/AndroidGradlePlugin.groovy

```diff
@@ -34,7 +34,7 @@ public abstract class AndroidGradlePlugin extends JavaBaseGradlePlugin {
 
 		android.defaultConfig {
 			minSdkVersion jdroid.minimumSdkVersion
-			targetSdkVersion jdroid.getIntegerProp('ANDROID_BUILD_TOOLS_VERSION', ANDROID_SDK_VERSION)
+			targetSdkVersion jdroid.getIntegerProp('ANDROID_COMPILE_SDK_VERSION', ANDROID_SDK_VERSION)
```

Comment:
It should be ANDROID_TARGET_SDK_VERSION, so any plugin user can configure different compile and target versions.